gramps_webapi/api/resources/emit.py

```diff
@@ -140,4 +140,7 @@ def default(self, obj: Any):
         if isinstance(obj, DbBookmarks):
             return self.api_filter(obj)
 
-        return JSONEncoder.default(self, obj)
+        try:
+            return JSONEncoder.default(self, obj)
+        except TypeError:
+            return None
```

Comment:
When is this expected to happen? I think it would be good to add a comment.

Reply:
Well, it should not be expected to happen, but in playing with filters I stumbled into a situation where some PyGObject got passed into emit when I tried to retrieve the filter list and I couldn't track down the root cause. So this insures should something like that happen things don't break completely.

Reply:
OK, but if it's un unexpected error it might make sence to at least log it to allow debugging (`current_app.logger.error(...)`).

Reply:
Good point, can I add that in next PR or want it in this one?

Reply:
As you prefer.